daemon/MenuDaemon.vala

```diff
@@ -48,6 +49,15 @@ namespace Gala {
         Gtk.MenuItem move_right;
         Gtk.MenuItem close;
 
+        // Desktop Menu
+        private Granite.AccelLabel change_wallpaper_accellabel;
+        private Granite.AccelLabel display_settings_accellabel;
+        private Granite.AccelLabel system_settings_accellabel;
+        Gtk.Menu? desktop_menu = null;
+        Gtk.MenuItem change_wallpaper;
+        Gtk.MenuItem display_settings;
+        Gtk.MenuItem system_settings;
```

Comment:
It doesn't appear that these are ever modified outside of the method that constructs them, so they probably don't need to be accessible by the whole class. You could construct them with `var foo`